### DISCUSSION TITLE
log reader loading all logs

### DISCUSSION THREAD

**Discussion by TechSupport26** [nextcloud/all-in-one#5640] [Questions] [answered]
From what I can tell. When you open the logging section in the admin page all of the logs get loaded.

I find this to be an issue as the page will freeze a lot until they are loaded. This also takes up quite a bit of memory depending on the amount of logs that you have.

Is the page intended to load so many logs and if so, would it be possible to add some kind of limit or page system where you can choose how many logs are displayed per page. E.g 10 or 100000

Maybe there is a setting somewhere that I have missed.
I am aware that you can filter log levels.

Thanks.

**Comment by szaimen** [comment, marked as answer]
Hi, please post this at https://help.nextcloud.com. Thanks!

**Reply by TechSupport26** [reply]
Thanks. I found an issue for it [here](https://github.com/nextcloud/logreader/issues/719)
